### PR TITLE
enhance async samples

### PR DESCRIPTION
**Pull request by beiwei30**

### CODE REVIEW

**Review by zouyx — APPROVED** [review]

File: async/README.md [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
+
+Dubbo-go provides not only synchronous invocation, but also asynchronous invocation on the consumer side. In order to
+use it, the client needs to implement the following interface to asynchronously receive the response from the service
+provider:
 
 ```golang
-//AsyncCallbackService callback interface for async
 type AsyncCallbackService interface {
-	CallBack(response CallbackResponse) // callback
+CallBack(response CallbackResponse)
```
Comment (AlexStocks) [inline async/README.md:11]: why delete the blank and the comment here?

File: async/go-client/pkg/user.go [hunk 2]
```diff
@@ -18,11 +18,13 @@
 package pkg
 
 import (
-	"context"
 	"github.com/apache/dubbo-go/common"
 	"github.com/apache/dubbo-go/protocol"
 	"github.com/apache/dubbo-go/remoting"
 	gxlog "github.com/dubbogo/gost/log"
+)
+import (
```
Comment (AlexStocks) [inline async/go-client/pkg/user.go:26]: hey, guy, move this import block above as the first import block.

File: async/README.md (hunk 1 above)
Comment (beiwei30) [reply async/README.md:11]: it's redundant, I think.

Comment (beiwei30) [reply async/README.md]: move back.

File: async/go-client/pkg/user.go (hunk 2 above)
Comment (beiwei30) [reply async/go-client/pkg/user.go:26]: fixed.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**